postgresql/db-statio-status.yml

```diff
@@ -0,0 +1,79 @@
+type: db
+data:
+  query: SELECT * FROM pg_statio_user_tables
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+
+observation:
+  - name: statio
+    metricNamespace: postgresql
+
+    metric:
+      - name: statio.table
+        source: relname
+        type: text
+        send: false
+        label: Table name
+
+      - name: statio.schema
+        source: schemaname
+        type: text
+        send: false
+        label: Schema name
+
+      - name: statio.heap.blocks.read
```

Comment:
How about `table.io.heap.blocks.read`? Same for other metrics in this file.

Reply:
Change to statio.heap.blocks.read